src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/EncryptedXml.cs

```diff
@@ -82,12 +80,9 @@ public class EncryptedXml
         //
         public EncryptedXml() : this(new XmlDocument()) { }
 
-        public EncryptedXml(XmlDocument document) : this(document, null) { }
-
-        public EncryptedXml(XmlDocument document, Evidence evidence)
```

Comment:
On the one hand, I think the type exists (System.Security.Permissions).  On the other hand, it's dead code in this implementation... so... I'm content.  We can add it back later if we decide that we need 100% API compatibility.